### DISCUSSION TITLE
Is it possible to capture the standardoutput stream so that I can get the progress percentage of the command line procedure?

### DISCUSSION THREAD

**Discussion by citystrawman** [Tyrrrz/CliWrap#234] [Q&A]
Hi, I am using CliWrap to run a commandline for my wpf project.  
The commandline runs about 40 minutes, and it will pop out standard output stream (although they does not give out the actual progress percentage, I can infer from its keywords)

My question is that, can I capture these standard output stream so that based on the keywords that I captured, I can get the progress percentage of this procedure and then I can give out this information so that it will be more user-friendly?

**Comment by Tyrrrz** [comment]
Yes.

https://github.com/Tyrrrz/CliWrap#pull-based-event-stream
